sdk/attestation/azure-security-attestation/src/attestation_client.cpp

```diff
@@ -46,6 +46,11 @@ AttestationClient::AttestationClient(
   m_apiVersion = options.Version.ToString();
   std::vector<std::unique_ptr<HttpPolicy>> perCallpolicies;
 
+#if defined(BUILD_TRANSPORT_WINHTTP_ADAPTER)
+  // This configuration will make winHTTP to disable client certificate for all attestation requests
+  perCallpolicies.emplace_back(std::make_unique<SetNoClientCertificatePolicy>());
+#endif
```

Comment:
@JeffreyRichter what are your thoughts on using transport-adapter specific pipeline policy within the client, to pass in a boolean via the context.

We can't use a transport option to configure this, since the customer shouldn't need to require knowledge of a particular service and which option to set.

Do we want to continue enabling customers to use the same adaptor across different clients and services?

Reply:
All Transport adapters would have to honor this accurately. We know that WinHTTP can do this. What about Curl?